tests/test_client.py

```diff
@@ -964,6 +966,37 @@ def test_bulk_get_documents(swg_index_client, swg_bulk_client):
         assert doc['did'] in dids
 
 
+def test_bulk_get_latest_version(swg_index_client, swg_bulk_client):
```

Comment:
can you split the tests with null/not-null into two? and also don't add latest versions for all docs, but rather to a subset and make sure that these cases also pass.

Reply:
ok so I'm gonna parametrize this, since I want to test get latest version including null with dids with or without null version, and same applies for get latest version excluding null.